src/main/java/com/zalando/ecommerce/controller/CartController.java

```diff
@@ -31,4 +35,44 @@ public ResponseEntity<?> getCart(@AuthenticationPrincipal UserDetails principal)
                     .body(new ErrorResponse(HttpStatus.NOT_ACCEPTABLE, e.getMessage()));
         }
     }
+
+    @PostMapping("/add")
```

Comment:
remove /add and this alone will do. 
```suggestion
    @PostMapping
```

Reply:
done 👍